src/raftstore/store/peer.rs

```diff
@@ -943,7 +943,8 @@ impl Peer {
             Ok(RequestPolicy::ProposeNormal) => self.propose_normal(req, metrics),
             Ok(RequestPolicy::ProposeTransferLeader) => {
                 self.propose_transfer_leader(req, cb, metrics);
-                return false;
+                // return true to let raftstore send out the message immediately.
```

Comment:
change to
// Make the Raft messages to be sent quickly if the transfer leader request is accepted.